--- libfprint-0.8.2/doc/meson.build.orig	2018-07-17 13:49:47.000000000 +0200
+++ libfprint-0.8.2/doc/meson.build	2018-12-16 15:40:04.836362925 +0100
@@ -54,7 +54,6 @@
              content_files: content_files,
              expand_content_files: expand_content_files,
              scan_args: [
-               '--rebuild-types',
                '--ignore-decorators=API_EXPORTED',
                '--ignore-headers=' + ' '.join(private_headers),
              ],
