--- libfprint-v1.94.10/libfprint/drivers/elan.h~	2026-02-10 00:00:00.000000000 +0100
+++ libfprint-v1.94.10/libfprint/drivers/elan.h	2026-02-10 00:00:01.000000000 +0100
@@ -30,9 +30,10 @@
 #define ELAN_0907 (1 << 0)
 #define ELAN_0C03 (1 << 1)
 #define ELAN_0C42 (1 << 2)
+#define ELAN_0C46 (1 << 3)

 /* devices which don't require frame rotation before assembling */
-#define ELAN_NOT_ROTATED ELAN_0C03
+#define ELAN_NOT_ROTATED ELAN_0C03 | ELAN_0C46

 /* min FW version that supports calibration */
 #define ELAN_MIN_CALIBRATION_FW 0x0138
@@ -217,6 +218,7 @@
   {.vid = ELAN_VEND_ID,  .pid = 0x0c42, .driver_data = ELAN_0C42},
   {.vid = ELAN_VEND_ID,  .pid = 0x0c4b, .driver_data = ELAN_ALL_DEV},
   {.vid = ELAN_VEND_ID,  .pid = 0x0c4d, .driver_data = ELAN_ALL_DEV},
+  {.vid = ELAN_VEND_ID,  .pid = 0x0c46, .driver_data = ELAN_0C46},
   {.vid = ELAN_VEND_ID,  .pid = 0x0c4f, .driver_data = ELAN_ALL_DEV},
   {.vid = ELAN_VEND_ID,  .pid = 0x0c63, .driver_data = ELAN_ALL_DEV},
   {.vid = ELAN_VEND_ID,  .pid = 0x0c6e, .driver_data = ELAN_ALL_DEV},
